ChildProject/projects.py

```diff
@@ -27,7 +38,7 @@ class ChildProject:
     :type path: str
     :param enforce_dtypes: enforce dtypes on children/recordings dataframes, defaults to False
     :type enforce_dtypes: bool, optional
-    :param ignore_discarded: ignore entries such that discard=1, defaults to False
+    :param ignore_discarded: ignore entries such that discard=1, defaults to True
```

Comment:
wow! big change of default!

Reply:
Yes and no, the usage of the `discard` flag is really scarce into our current datasets. Especially as the existing code was bugged so basically it was behaving opposite of what it should have done. The fix to this and that change of default behavior will hopefully open the door to using more consistently the discard flag to ignore edge case audios (like tests, too small etc) rather than removing them from the dataset and losing all trace of their existence.